bundle/manifests/oadp-operator.clusterserviceversion.yaml

```
@@ -394,8 +394,14 @@ spec:
       kind: CloudStorage
       name: cloudstorages.oadp.openshift.io
       statusDescriptors:
-      - description: LastSyncTimestamp is the last time the contents of the CloudStorage
-          was synced
+      - description: Conditions represent the latest available observations of the
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)